deployment/osm-stats.tf

```diff
@@ -89,288 +30,152 @@ resource "azurerm_container_group" "osm-stats" {
 
     environment_variables = {
       DATABASE_URL = "postgresql://${var.db_user}%40${var.db_server_name}:${random_string.db_password.result}@${azurerm_postgresql_server.osm-stats.fqdn}/${var.db_name}"
-      OVERPASS_URL="http://export.hotosm.org:6080"
+      /**
+      DATABASE_URL = format(
+        "postgresql://%s%%40%s:%s@%s/%s",
+        var.db_user,
+        var.db_server_name,
+        random_string.db_password.result,
+        azurerm_postgresql_server.osm-stats.fqdn,
+        var.db_name
+      ) **/
+      OVERPASS_URL = "http://export.hotosm.org:6080"
```

Comment:
Can you help me understand these variables? Why is the OVERPASS_URL hardcoded?

Reply:
It shouldn't be. I parameterized it.